aspnetcore/includes/blazor-preview-notice.md

```diff
@@ -0,0 +1,6 @@
+> [!WARNING]
```

Comment:
I think I prefer IMPORTANT. Or is there just a generic NOTE?

Reply:
Yes, IMPORTANT is purple: https://docs.microsoft.com/en-us/aspnet/core/client-side/bower

NOTE is blue: https://docs.microsoft.com/en-us/aspnet/core/client-side/

I'll make it a NOTE.